src/emerge.cpp

```diff
@@ -600,6 +600,7 @@ EmergeAction EmergeThread::getBlockOrStartGen(
 		if ((*block)->isGenerated())
 			return EMERGE_FROM_MEMORY;
 	} else {
+		ScopeProfiler sp(g_profiler, "EmergeThread: load block", SPT_AVG);
```

Comment:
does this intentionally count the time for deserialization?

Reply:
Yes. Although one can argue that.

I should also point out that is _attempted_ loads. The block might have to generated, in which case we look it up in the database but won't find it there.

Still. I found this quite helpful debugging all the settings I mentioned above.